config/config-example.yaml

```diff
@@ -4,27 +4,236 @@ receivers:
     collection_interval: 1m
     scrapers:
       memory:
```

Comment:
I would prefer if these were in alphabetical order 😄  (other than `process` that I usually put at the end since it is kind of a special case and a lot more expensive than the other scrapers)